spec.bs

```diff
@@ -721,32 +774,72 @@ To <dfn>create a request</dfn> given a [=URL=] |url|, a [=string=] <dfn for="cre
 1. Return |request|.
 
 <div algorithm>
+To <dfn>validate fetching response</dfn> given a [=response=] |response| and |responseBody|, and a

```

Comment:
Should IG updates also use [=validate fetching response=]?

Reply:
Yes, done.